WORKSPACE

```diff
@@ -103,6 +103,27 @@ stack_snapshot(
     },
 )
 
+new_local_repository(
```

Comment:
@googleson78 do you have any idea of a better / more portable way to provide the library to the test? Unfortunately it has a bunch of dependencies https://github.com/google/libphonenumber/tree/master/cpp. I don't know much about nix, but maybe there is a nix package for it? (https://github.com/google/libphonenumber/tree/master/cpp)